target/linux/ramips/image/mt7620.mk

```diff
@@ -489,21 +489,12 @@ endef
 TARGET_DEVICES += hnet_c108
 
 define Device/hootoo_ht-tm05
+  $(Device/sunvalley_loader_okli)
   SOC := mt7620n
   IMAGE_SIZE := 6144k
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci kmod-i2c-ralink
```

Comment:
move this right after "DEVICE_MODEL" so that it's not seen as a line change

Reply:
Good point, done - thanks!